blocks/top-table-list-block/features/top-table-list/_children/medium-list-item.test.jsx

```diff
@@ -106,77 +106,77 @@ describe('medium list item', () => {
-  //   const secondaryFont = 'Georgia';
-  //   const by = ['jack'];
-  //   const element = { credits: { by: [] } };
-  //   const displayDate = '';
-  //   const id = 'test';
-  //   const { default: MediumListItem } = require('./medium-list-item');
-
-  //   // eslint-disable-next-line no-unused-vars
-  //   const wrapper = mount(<MediumListItem
-  //     imageURL={imageURL}
-  //     constructedURL={constructedURL}
-  //     itemTitle={itemTitle}
-  //     descriptionText={descriptionText}
-  //     primaryFont={primaryFont}
-  //     secondaryFont={secondaryFont}
-  //     by={by}
-  //     element={element}
-  //     displayDate={displayDate}
-  //     id={id}
-  //     customFields={headBelowConfig}
-  //   />);
-
-  //   expect(wrapper.find('.headline-below').length).toBe(1);
-  //   expect(wrapper.find('.headline-above').length).toBe(0);
-  // });
+  it('headline has class headline-above when headline position is above', () => {
+    const imageURL = 'pic';
+    const constructedURL = 'url';
+    const itemTitle = 'title';
+    const descriptionText = 'description';
+    const primaryFont = 'arial';
+    const secondaryFont = 'Georgia';
+    const by = ['jack'];
+    const element = { credits: { by: [] } };
+    const displayDate = '';
+    const id = 'test';
+    const { default: MediumListItem } = require('./medium-list-item');
+
+    // eslint-disable-next-line no-unused-vars
+    const wrapper = mount(<MediumListItem
+      imageURL={imageURL}
+      constructedURL={constructedURL}
+      itemTitle={itemTitle}
+      descriptionText={descriptionText}
+      primaryFont={primaryFont}
+      secondaryFont={secondaryFont}
+      by={by}
+      element={element}
+      displayDate={displayDate}
+      id={id}
+      customFields={config}
+    />);
+
+    expect(wrapper.find('.headline-above').length).toBe(1);
+    expect(wrapper.find('.headline-below').length).toBe(0);
+  });
+
+  it('headline has class headline-below when headline position is below', () => {
+    const imageURL = 'pic';
+    const constructedURL = 'url';
+    const itemTitle = 'title';
+    const descriptionText = 'description';
+    const primaryFont = 'arial';
+    const secondaryFont = 'Georgia';
+    const by = ['jack'];
+    const element = { credits: { by: [] } };
+    const displayDate = '';
+    const id = 'test';
+    const { default: MediumListItem } = require('./medium-list-item');
+
+    // eslint-disable-next-line no-unused-vars
+    const wrapper = mount(<MediumListItem
+      imageURL={imageURL}
+      constructedURL={constructedURL}
+      itemTitle={itemTitle}
+      descriptionText={descriptionText}
+      primaryFont={primaryFont}
+      secondaryFont={secondaryFont}
+      by={by}
+      element={element}
+      displayDate={displayDate}
+      id={id}
+      customFields={headBelowConfig}
+    />);
+
+    expect(wrapper.find('.headline-below').length).toBe(1);
+    expect(wrapper.find('.headline-above').length).toBe(0);
```

Comment:
this is a good test -- I like it! shows the previous case of headline below being 1 and the bad case headline above being not shown